src/app/merge-diff/merge-diff.module.ts

```diff
@@ -22,10 +22,13 @@ import { CommonModule } from '@angular/common';
 import { SharedModule } from '@mdm/modules/shared/shared.module';
 import { MaterialModule } from '@mdm/modules/material/material.module';
 import { MergeDiffContainerComponent } from '@mdm/merge-diff/merge-diff-container/merge-diff-container.component';
+import { ResizableDirective } from '@mdm/directives/resizable.directive';
+
 
 @NgModule({
   declarations: [
-    MergeDiffContainerComponent
+    MergeDiffContainerComponent,
+    ResizableDirective
```

Comment:
Fine here for now, but I imagine this could be moved to a more common module such as `SharedModule`